src/content/troubleshooting/guides/build-tools-troubleshooting/specific-cli-version.md

```diff
@@ -39,11 +40,22 @@ npm install -g particle-cli@2
 To restore the current version of the Particle CLI you can use:
 
 ```
+npm uninstall -g particle-cli
 particle --update-cli
 ```
 
-Also note that the Particle CLI current requires node v16. It does not work with node v18 and later. Using a node version manager like nvm or fnm is recommended so you can switch between version if necessary.
+Also note that the Particle CLI current requires node v16. It does not work with node v18 and later. Using a node version manager like nvm or fnm is recommended so you can switch between version if necessary. This is only necessary when using a source build, not when using the Particle CLI installer.
 
-## Mac with Apple silicon
+### Mac with Apple silicon
 
-If you are using a Mac with Apple silicon (M1, M2, M3, ...) and have previously imported your settings and applications from an Intel Mac see [CLI on Mac with Apple silicon](/troubleshooting/guides/build-tools-troubleshooting/cli-mac-apple-silicon/) for additional information.
+If you are using a Mac with Apple silicon (M1, M2, M3, ...) and have previously imported your settings and applications from an Intel Mac see [CLI on Mac with Apple silicon](/troubleshooting/guides/build-tools-troubleshooting/cli-mac-apple-silicon/) for additional information. These steps should not be necessary if using the CLI installer.
+
+{{collapse op="end"}}
+
+## Windows installer historical versions
+
+The link above will download the latest version of the Windows installer for the Particle CLI. If for some reason you need to install and older
+version you can download previous versions here. We recommend that you always install the latest version, as there is no longer any association
+between CLI and version and Device OS versions.
+
+{{> windows-cli-downloads}}
```

Comment:
Can you add a section on Mac and Linux older versions, something like:

```
## Mac OS or Linux installer historical version

To install a specific version of the CLI on Mac OS or Linux, specify the `VERSION` environment variable.

VERSION=3.23.4 bash <( curl -sL https://particle.io/install-cli )

Check the table of available version in the Windows installer historical versions.


```

Reply:
Done. I assume there should be a `&&` in the bash command line, like:

```
VERSION=3.23.4 && bash <( curl -sL https://particle.io/install-cli )
```